jetty-ee10/jetty-ee10-maven-plugin/src/main/java/org/eclipse/jetty/ee10/maven/plugin/AbstractWebAppMojo.java

```diff
@@ -48,8 +48,12 @@
 import org.apache.maven.project.MavenProject;
 import org.codehaus.plexus.util.StringUtils;
 import org.eclipse.aether.RepositorySystem;
-import org.eclipse.jetty.ee10.maven.plugin.utils.MavenProjectHelper;
 import org.eclipse.jetty.ee10.servlet.security.LoginService;
+import org.eclipse.jetty.maven.MavenServerConnector;
```

Comment:
correct I haven't finished everything yet and it's more removal coming soon ;)

Reply:
> This class needs to extend `AbstractWebAppMojo` from `jetty-maven-core` doesn't it?

nah we can't because each single `AbstractWebAppMojo` from every eeX is using `org.eclipse.jetty.eeX.security.LoginService;` 
I'm very tempted to removed this field which is marked as deprecated as users can simply user a `jetty-web.xml` for this